src/napari_epitools/widgets/projection.py

```diff
@@ -24,11 +24,8 @@ def create_projection_widget() -> magicgui.widgets.Container:
         value=1,
         name="smoothing_radius",
         label="smoothing radius",
-        widget_type="FloatSlider",
+        widget_type="FloatSpinBox",
         options={
-            "min": 0,
-            "max": 10,
-            "step": 0.1,
```

Comment:
Why remove these? Are they the default values?

Reply:
these parameters were needed to set the range limits and step sizeof the `Slider` widgets. But now we've switched to `SpinBox` widgets we can just use the default values for the range and step (the range is something like -1000 to 1000)

Reply:
Okay!